src/java/org/apache/cassandra/service/accord/txn/TxnReference.java

```diff
@@ -18,29 +18,21 @@
 
 package org.apache.cassandra.service.accord.txn;
 
-import java.io.IOException;
-import java.nio.ByteBuffer;
-import java.util.Objects;
-
 import org.apache.cassandra.db.TypeSizes;
-import org.apache.cassandra.db.marshal.AbstractType;
-import org.apache.cassandra.db.marshal.ByteBufferAccessor;
-import org.apache.cassandra.db.marshal.CollectionType;
-import org.apache.cassandra.db.marshal.SetType;
-import org.apache.cassandra.db.marshal.UserType;
+import org.apache.cassandra.db.marshal.*;
 import org.apache.cassandra.db.partitions.FilteredPartition;
-import org.apache.cassandra.db.rows.Cell;
-import org.apache.cassandra.db.rows.CellPath;
-import org.apache.cassandra.db.rows.ColumnData;
-import org.apache.cassandra.db.rows.ComplexColumnData;
-import org.apache.cassandra.db.rows.Row;
+import org.apache.cassandra.db.rows.*;
 import org.apache.cassandra.io.IVersionedSerializer;
 import org.apache.cassandra.io.util.DataInputPlus;
 import org.apache.cassandra.io.util.DataOutputPlus;
 import org.apache.cassandra.schema.ColumnMetadata;
 import org.apache.cassandra.transport.ProtocolVersion;
 import org.apache.cassandra.utils.ByteBufferUtil;
 
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Objects;
```

Comment:
TODO: restore import order/revert wildcards

Reply:
We don't have a threshold for when to use wildcards. Depending on the setting it will either remove wildcard imports or add wildcard imports.

I don't see an option to avoid fighting on that particular issue, even something like "don't change untouched imports"